src/PctvData.cpp

```diff
@@ -935,6 +935,7 @@ PVR_ERROR Pctv::GetCapabilities(kodi::addon::PVRCapabilities& capabilities)
   capabilities.SetSupportsRadio(false);
   capabilities.SetSupportsChannelGroups(true);
   capabilities.SetSupportsRecordings(true);
+  capabilities.SetSupportsRecordingsDelete(false);
```

Comment:
Should this be false?

Reply:
Should be `false` as there no `DeleteRecording()` recording used inside him.